### PR TITLE
[Expired sessions] Fix query

### PR DESCRIPTION
**Pull request by malted**

### CODE REVIEW

**Review by sampoder — DISMISSED** [review]
lgtm!

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -369,10 +369,8 @@ def edit_security
                         .user_sessions
                         .only_deleted
```
Comment (garyhtou) [inline app/controllers/users_controller.rb]: I think the `only_deleted` will hide sessions that expired because of time. Maybe we want `with_deleted`

Comment (malted) [reply app/controllers/users_controller.rb]: wait this is a great catch

**Review by garyhtou — APPROVED** [review]
There's one small bug left: Expired sessions older than 1 week may show up

**Pull request merged**